init.lua

```diff
@@ -48,7 +48,7 @@ function obj:init()
         self.controlKeyTimer:start()
       else
         if self.sendEscape then
-          hs.eventtap.keyStroke({}, 'escape')
+          hs.eventtap.keyStroke({}, 'escape', 0)
```

Comment:
```suggestion
          hs.eventtap.keyStroke({}, 'escape', 1)
```

Using `1` works with [issue 8](https://github.com/jasonrudolph/ControlEscape.spoon/pull/8#issue-413043363) and should solve your issues as well.

Reply:
can we please accept this suggestion and merge it?